src/services/logAnalyticsWorkspace/format.ts

```diff
@@ -18,9 +18,13 @@ export default ({
     resourceGroupId,
     provisioningState,
     customerId,
-    sku,
-    features,
-    workspaceCapping,
+    sku: { name, lastSkuUpdate },
```

Comment:
do these need default values?

Reply:
Not really. Although it is optional on the type, both properties are always populated.